elyra/pipeline/processor.py

```diff
@@ -116,6 +103,7 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.root_dir = get_expanded_path(kwargs.get('root_dir'))
         self._registry = PipelineProcessorRegistry.instance()
+        self._component_catalog = ComponentCatalog.instance(parent=self.parent)
```

Comment:
We don't really need the variable here since it isn't currently accessed anywhere else in the class - I just followed the pattern that the `PipelineProcessorRegistry` took

Reply:
Removed in the latest set of commits